packages/common/src/core/types/IHubLocation.ts

```diff
@@ -0,0 +1,26 @@
+import { ISpatialReference } from "@esri/arcgis-rest-types";
+
+export interface IHubLocation {
```

Comment:
jsdoc comment?

Reply:
Also - please run `npm run docs:serve` and verify that these property comments are included - I'm not 100% sure if they pick up single-line comments `//` vs the `/** ...` style